pyproject.toml

```diff
@@ -40,10 +40,12 @@ authors = [
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.7.4"
```

Comment:
We're using Python 3.9 for the Lambda runtime, so we could just use that here as well. What do you think?